### DISCUSSION TITLE
Enhancer for YouTube

### DISCUSSION THREAD

**Discussion by MrDowntempo** [catppuccin/catppuccin#2064] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

I'm wary of using extensions in firefox unless I absolutely need them, so I've not been using stylus. However, YouTube has frustrated me with its pushing shorts on my to no end, so I've resorted to using what seems to be the most popular extension to hide them, Enhancer for YouTube. It has many other features too, including the ability to add a custom theme. It would be wonderful if there was a premade Catpuccin theme to copypasta in there. Effectively it should appear very similar to the way the stylus theme for YouTube works.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by anoraktrend** [comment]
it would be even better if the [mrfdev](webmaster@mrfdev.com) could incorporate the existing themes in the app.

**Comment by anoraktrend** [comment]
however if you want something basic via the custom deepdark mode:
```
(@accent color) 
(@base)
(@surface0)
(@surface1)
(@text)
(@subtext1)
(@surface2)
```
copy paste the hexcode into the boxes in this order for the specified @ tags for the specific theme you want to use.

**Reply by anoraktrend** [reply]
though, you might be better off looking at the stylus userstyle.
